activerecord/test/schema/schema.rb

```diff
@@ -169,7 +169,7 @@
   end
 
   create_table :comments, force: true do |t|
-    t.integer :post_id, null: false
```

Comment:
My worry is that a test is assuming it's there, for the purposes of proving that something works with a `null: false` column. So removing it wouldn't cause a test to fail, but may cause a test to stop testing what it previously was.